charts/camunda-platform-alpha/templates/optimize/configmap.yaml

```diff
@@ -23,6 +23,11 @@ data:
 
     {{- if .Values.global.elasticsearch.enabled }}
     es:
+      settings:
+        {{- if .Values.core.index.prefix }}
+        index:
+          prefix: {{ .Values.core.index.prefix | quote }}
+        {{- end }}
```

Comment:
❌ If we use for the ES exporter different prefix, then likely we need to use here the same for optimize

Reply:
Fixed in: https://github.com/camunda/camunda-platform-helm/commit/ae9c82512781c1c307eb20a96589d0b7575aa3e0